ansible/avena/roles/core/files/systemd/container-maintainer.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Run container-maintainer.service every day
+Requires=container-maintainer.service
+
+[Timer]
+Unit=container-maintainer.service
+OnCalendar=daily
```

Comment:
Let's set a time of day so we know when it should have been done.

Reply:
Understood. Currently it is at midnight meaning that will be done right when the ISOBlue boots in the morning